qiita_pet/static/js/qiita.js

```diff
@@ -314,17 +314,20 @@ function format_biom_rows(data, row, for_study_list = true, samples = null) {
 
   // grouping by processing_method, data_type and parameters
   $.each(data, function (idx, info) {
-    if (typeof info !== 'string' && !(info instanceof String)) {
-      var algorithm = info.algorithm;
-      if (!(algorithm in processing_method)) {
-        processing_method[algorithm] = {};
-      }
+    // ignore the artifacts that were generated with software that is not active
+    if (info['active']) {
+      if (typeof info !== 'string' && !(info instanceof String)) {
+        var algorithm = info.algorithm;
+        if (!(algorithm in processing_method)) {
+          processing_method[algorithm] = {};
+        }
 
-      var data_type = info.data_type + ' (' + info.target_subfragment.join(', ') + ')';
-      if (!(data_type in processing_method[algorithm])) {
-        processing_method[algorithm][data_type] = [];
+        var data_type = info.data_type + ' (' + info.target_subfragment.join(', ') + ')';
+        if (!(data_type in processing_method[algorithm])) {
```

Comment:
Is this checking that data_type is a substring of processing_method.algorithm?

Reply:
yup, basically this is grouping the artifacts per data_type and it doesn't exits, it creates it ... a classic:
```
myvar = {}
for x in y:
   if x not in myvar:
      myvar[x] = init
  .....
```

Reply:
I see, thanks!